src/init.cpp

```diff
@@ -295,7 +295,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-conf=<file>", strprintf(_("Specify configuration file (default: %s)"), "bitcoin.conf"));
     if (mode == HMM_BITCOIND)
     {
-#if !defined(WIN32)
+#ifndef WIN32
```

Comment:
I just don't see it as very constructive - e.g. a) a lot of pull requests hit this code due to adding/removing arguments, and b) there's an infinite number of commits like this possible, the next person may change them to `#if !` again for aesthetic reasons. 

But no need to revert it, the other changes in this pull are useful, so I'm just going to merge...